src/components/LetniaHome.vue

```diff
@@ -1,19 +1,52 @@
 <template>
   <div class="container">
     <h3>Letnia kolonia</h3>
-    <h3>Siemanko</h3>
+    <h1 class="tooltip-inner">Beta testing scss</h1>
+   
+
+    <template>
+        <div>
+           <button @click="changeColor">Zmień kolor</button>
+        </div>
+    </template>
+
+
+
+
+
+
+
   </div>
 </template>
 
+<script>
```

Comment:
We already use script element in this component. Please move this new code to already existing script element